### DISCUSSION TITLE
We hit 2K followers, yay!

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#139] [Show and tell]
Thank you to everyone who believes in the Fossify project.

> ...I would be happy to do so in my free time...

I didn't think me wanting to maintain a fork would get this much attention and approval yet here we are. It turns out, the project demands my full-time attention and I hope to start full-time once there is some way to fund this "crusade". (I'm aware of the grants and donations offered by various organizations (application sent) but eventually, the project must sustain itself to be long lasting).

The more people know about this project, the higher the chances that Fossify will succeed so please share the project with people who care about FOSSifying things. 

Special thanks to @Aga-C for being the most helpful, @ronniedroid for the website, and all the other contributors and people who donated to the cause.

**Comment by ronniedroid** [comment]
I am glad I could/can help, there is still a lot of work left and hopefully I will be free again soon to work on it. 

Hopefully Fossify becomes sustainable soon, thank you a lot for your work.

**Comment by siccovansas** [comment]
More good news, [FossifyOrg](https://github.com/orgs/FossifyOrg) (2086) now has more followers than [SimpleMobileTools](https://github.com/SimpleMobileTools) (2078) :rocket::muscle:!

    curl -H "Accept: application/json" https://api.github.com/orgs/FossifyOrg | jq ".followers
    curl -H "Accept: application/json" https://api.github.com/orgs/SimpleMobileTools | jq ".followers

Thanks for all the work @naveensingh and others for making the fork already this popular! I really hope there will be enough funds for you to start full-time :pray:. I've setup a recurring donation and hope others who can will [contribute](https://github.com/FossifyOrg#support-fossify-heart) as well.

**Reply by naveensingh** [reply]
Thank you :)

**Comment by Mr-Bajs** [comment]
@naveensingh You might should consider making a blog? I find them useful for any project that I follow, it's a good way to keep up with project and help to set the expectaions right. Also new media, nerds and forums often use blog post as a new article. It's a great way to comminicate to the users, it's also a indirect way of marketing.

**Reply by naveensingh** [reply]
Yes, there will be a blog and there will be more social media accounts. It's not up already only because I'm mostly focusing on getting the apps released.

**Reply by ronniedroid** [reply]
The blog will be implemented next week, expect an update to the PR on Sunday or Monday.

**Reply by Mr-Bajs** [reply]
Great. I assume it will be RSS-combatible, whenever it is done. :)

**Comment by antodc** [comment]
Actually, you got 4k followers now!!
Many congrats to @naveensingh, @Aga-C and all the contributors!

**Reply by naveensingh** [reply]
The phoenix is rising ;)

**Reply by pandaroux007** [reply]
**Well done!** I find the idea of ​​the project incredible and it's exactly what I've been looking for for a long time, I don't think I'm the only one - very happy to learn that people like @naveensingh or @Aga-C had the idea to take up the torch so that these apps don't get lost! **Congratulations to you, really!**

(_besides, how is the project going now?_)

**Reply by naveensingh** [reply]
_It's going good._
